community/mg/sources

```diff
@@ -0,0 +1 @@
+git+https://github.com/hboetes/mg.git
```

Comment:
`.git` can be dropped from the end. Will also have to update checksums afterwards.

Reply:
Will fix.